src/models/generated/TwoTerminalLCCLine.jl

```diff
@@ -0,0 +1,398 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+
+#! format: off
+
+"""
+    mutable struct TwoTerminalLCCLine <: TwoTerminalHVDC
+        name::String
+        available::Bool
+        arc::Arc
+        active_power_flow::Float64
+        r::Float64
+        transfer_setpoint::Float64
+        scheduled_dc_voltage::Float64
+        rectifier_bridges::Int
```

Comment:
nitpick: I kind of expected `get_rectifier_bridges` to return a `Vector` of something. maybe `n_rectifier_bridges`? same for `inverter_bridges`

Reply:
we can make it more clear in the docstring but its is late for the rename of the field.